1_fetch/src/fetch_data.R

```diff
@@ -0,0 +1,6 @@
+# Function to fetch data
+fetch_data <- function(sb_id, item_name, out_dir, out_file) {
+  destination <- file.path(out_dir, out_file)
+  item_file_download(sb_id, names = item_name, destinations = destination, 
+                     overwrite_file = TRUE) 
+}
```

Comment:
You should add a `return()` as the last line within all functions, or a comment that says what is returned. For `targets`, it helps to return an output file path or an R object (that will be explained in upcoming training issues). File path works well here for the download step